### ISSUE TITLE
Incorrect caveat description provides wrong output (also differs from original repo)

### ISSUE THREAD

**Issue by ahmad-musleh** [Homebrew/homebrew-core#28879]
**Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.**

- [x] are reporting a bug others will be able to reproduce and not asking a question. If you're not sure or want to ask a question do so on our Discourse: https://discourse.brew.sh
- [x] have a problem with `brew install` (or `upgrade`, `reinstall`) a single, official formula (not cask)? If it's a general `brew` problem please file this issue at Homebrew/brew: https://github.com/Homebrew/brew/issues/new/choose. If it's a `brew cask` problem please file this issue at https://github.com/Homebrew/homebrew-cask/issues/new/choose. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [x] ran `brew update` and can still reproduce the problem?
- [ ] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [ ] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [ ] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

To help us debug your issue please explain:
- What you were trying to do (and why)
  - brew install kube-ps1
  - adding the code in caveats to .bash_profiles
- What happened (include command output)
Installation: 
```
==> Downloading https://github.com/jonmosco/kube-ps1/archive/0.6.0.tar.gz
==> Downloading from https://codeload.github.com/jonmosco/kube-ps1/tar.gz/0.6.0
######################################################################## 100.0%
==> Caveats
Make sure kube-ps1 is loaded from your ~/.zshrc or ~/.bashrc:
  For zsh:
  source "/usr/local/opt/kube-ps1/share/kube-ps1.sh"
  PROMPT='$(kube_ps1)'$PROMPT

  For Bash:
  source "/usr/local/opt/kube-ps1/share/kube-ps1.sh"
  PS1="[$(kube_ps1)]$ "
==> Summary
🍺  /usr/local/Cellar/kube-ps1/0.6.0: 6 files, 29.0KB, built in 5 seconds

```
Adding
```
source "/usr/local/opt/kube-ps1/share/kube-ps1.sh"
PS1="[$(kube_ps1)]$ "
``` 
to .bash_profiles gives me: 
```
[(⎈ |:)]$
```

- What you expected to happen
  - Replacing second line with `PS1='[\u@\h \W $(kube_ps1)]\$ '` from the original repo gives me:
```
ahmad (⎈ |minikube:default)]$
```
which seems to be the correct presentation.
- Step-by-step reproduction instructions (by running `brew install` commands)

**Comment by ilovezfs** [comment]
@jonmosco do you mind taking a look at fixing this please?

**Comment by apjanke** [comment]
Since `$PROMPT` is an alias for `$PS1` in `zsh`, the instructions for zsh and bash can probably be merged here. The appending pattern used by the zsh instructions looks like a better approach to me. I'll give this a whack when my computer frees up.

**Comment by apjanke** [comment]
Here's a PR to fix: #29133

This fixes the bash instructions by using the correct quotes (single quotes to delay evaluation) and having it prepend the kube-ps1 prompt to the user's existing prompt, instead of overwriting it.

Then it simplifies the instructions by merging the bash and zsh variants together, which can be done because zsh is bash-compatible enough that you can use the same code for both, as long as you use `$PS1`.

Could you test this out on your system and see if it works for you?

**Comment by ahmad-musleh** [comment]
Hi @apjanke 
It worked 😄 

Just to be sure. I replaced 
```source "#{opt_share}/kube-ps1.sh"```
 with what brew spit out originally which is 
```source "/usr/local/opt/kube-ps1/share/kube-ps1.sh"```
 in my case which I assume will be the case when the PR is merged. 

Thanks 👍
